cmd/edgectl/aes_install.go

```diff
@@ -437,166 +428,97 @@ func (i *Installer) Perform(kcontext string) error {
 		return err
 	}
 
-	// Allow overriding the source domain (e.g., for smoke tests before release)
-	manifestsDomain := "www.getambassador.io"
-	domainOverrideVar := "AES_MANIFESTS_DOMAIN"
-	overrideMessage := "Downloading manifests from override domain %q instead of default %q because the environment variable %s is set."
-	if amd := os.Getenv(domainOverrideVar); amd != "" {
-		i.ShowWrapped(fmt.Sprintf(overrideMessage, amd, manifestsDomain, domainOverrideVar))
-		manifestsDomain = amd
+	// Try to grab some cluster info
+	if err := i.UpdateClusterInfo() ; err != nil {
+		i.ShowWrapped(fmt.Sprintf("-> Failed to get some cluster information"), seeDocs)
+		i.Report("fail_cluster_info", ScoutMeta{"err", err.Error()})
+		return errors.Errorf("could not get cluster info")
 	}
+	i.SetMetadatum("Cluster Info", "cluster_info", i.clusterinfo.name)
 
-	// Download AES manifests
-	crdManifests, err := getManifest(fmt.Sprintf("https://%s/yaml/aes-crds.yaml", manifestsDomain))
-	if err != nil {
-		i.Report("fail_no_internet", ScoutMeta{"err", err.Error()})
-		return errors.Wrap(err, "download AES CRD manifests")
-	}
-	aesManifests, err := getManifest(fmt.Sprintf("https://%s/yaml/aes.yaml", manifestsDomain))
+	// create a new parsed checker for versions
+	chartVersion, err := helm.NewChartVersionRule("*")
 	if err != nil {
-		i.Report("fail_no_internet", ScoutMeta{"err", err.Error()})
-		return errors.Wrap(err, "download AES manifests")
+		return err
 	}
 
-	// Figure out what version of AES is being installed
-	aesVersionRE := regexp.MustCompile("image: quay[.]io/datawire/aes:([[:^space:]]+)[[:space:]]")
-	matches := aesVersionRE.FindStringSubmatch(aesManifests)
-	if len(matches) != 2 {
-		i.log.Printf("matches is %+v", matches)
-		i.Report("fail_bad_manifests")
-		return errors.Errorf("Failed to parse downloaded manifests. Is there a proxy server interfering with HTTP downloads?")
+	options := helm.HelmManagerOptions{
+		Version: chartVersion,
+		Logger:  i.log,
 	}
-	i.version = matches[1]
-	i.SetMetadatum("AES version being installed", "aes_version", i.version)
 
-	// Display version information
-	i.ShowWrapped(fmt.Sprintf("-> Installing the Ambassador Edge Stack %s.", i.version))
+	chartValues := map[string]string{} // TODO: set any Helm values we want to override,
```

Comment:
Does the chart install three replicas? Can we change that to one replica (and would that be done here)? I suggest this for consistency with the old installer, but maybe that's not a good reason. Thoughts?

Reply:
According to the [defaults in the Chart](https://github.com/datawire/ambassador-chart/#configuration), replicas is 3. We could add a map with the default chart values, and use `replicas=1`.

Reply:
Let's keep at 3, please.

Reply:
Okay, as long as we're aware that
- the old installer installed one replica because...
- the manual install YAML specifies one replica

Reply:
I didn't realize we changed it. Then let's do 1.

Reply:
Yup, I'm not sure why the OSS YAML and the Helm chart say 3, but the AES YAML says 1. But that's the way it has been since at least 1.0.